src/wallet/wallet.cpp

```diff
@@ -275,6 +275,11 @@ std::shared_ptr<CWallet> CreateWallet(WalletContext& context, const std::string&
         wallet_creation_flags |= WALLET_FLAG_BLANK_WALLET;
     }
 
+    // Legacy wallets are being deprecated, warn if a newly created wallet is legacy
+    if (!(wallet_creation_flags & WALLET_FLAG_DESCRIPTORS)) {
+        warnings.push_back(_("The Legacy wallet type is being deprecated. Support for opening Legacy wallets will be removed in the future"));
```

Comment:
- s/opening/creating/
- s/Legacy/legacy/
- be unambiguous that the wallet was still created

suggestion:
```suggestion
        warnings.push_back(_("Wallet successfully created, but be aware that the legacy wallet type is expected to be deprecated, and support for creating legacy wallets may be removed in the future"));
```

Reply:
Added `Wallet created successfully` and lowercase `legacy.` However there is no "expected to be" or "may" about the removal of the legacy wallet. It _will_ be removed in the future, whether that's in a few years, or in fifty. Additionally, this warning is to inform users that their newly created legacy wallet will have issues with being opened in the future. It's already created, so warning that creation will be removed is useless. `opening` is the correct word to use.

Reply:
Thanks!